tests/unit/amo/pages/TestUserProfileEdit.js

```diff
@@ -258,55 +270,52 @@ describe(__filename, () => {
     // called.
     store.dispatch(
       loadUserNotifications({
-        username,
+        userId,
         notifications: createUserNotificationsResponse(),
       }),
     );
 
-    const root = renderUserProfileEdit({ errorHandler, store });
+    const root = renderUserProfileEdit({ errorHandler, params, store });
     const user = getCurrentUser(store.getState().users);
 
     dispatchSpy.resetHistory();
 
     // We pass the `user` to simulate the case where the user data are already
     // present in the store.
-    root.setProps({ username: 'killmonger', user });
+    root.setProps({ userId: userId + 999, user });
 
     sinon.assert.notCalled(dispatchSpy);
   });
 
   it('fetches user notifications when not loaded yet', () => {
-    const username = 'tofumatt';
+    const userId = 34;
 
     // When loading the current user, their notifications are not loaded yet
     // and we should dispatch `fetchUserNotifications`.
-    const { store } = signInUserWithUsername(username);
+    const { params, store } = signInUserWithUserId(userId);
     const dispatchSpy = sinon.spy(store, 'dispatch');
     const errorHandler = createStubErrorHandler();
 
-    const root = renderUserProfileEdit({ errorHandler, store });
+    const root = renderUserProfileEdit({ errorHandler, params, store });
     const user = getCurrentUser(store.getState().users);
 
     dispatchSpy.resetHistory();
 
     // We pass the `user` to simulate the case where the user data are already
     // present in the store.
-    root.setProps({ username: 'killmonger', user });
+    root.setProps({ userId: userId + 999, user });
 
     sinon.assert.calledWith(
       dispatchSpy,
       fetchUserNotifications({
         errorHandlerId: errorHandler.id,
-        username,
+        userId,
       }),
     );
   });
 
   it('does not dispatch fetchUserAccount if username does not change', () => {
```

Comment:
Should this be "does not dispatch fetchUserAccount if *userId* does not change"?

Reply:
yep